wai-videos/video-introduction/video-introduction.id.vtt

```diff
@@ -97,19 +97,19 @@ dan diadopsi sebagai standar di Eropa dengan nama EN 301 549.
 WCAG dibangun berdasarkan empat prinsip utama:
 
 00:02:33.560 --> 00:02:39.920 align:middle
-Pertama, Dapat Dipersepsikan - sebagai contoh, seseorang bisa melihat sebuah konten, atau mendegarnya.
+Pertama, dapat diterima indra - sebagai contoh, seseorang bisa melihat sebuah konten, atau mendegarnya.
```

Comment:
@frirasyidi Because this translates a key WCAG principle, I leave it for discussion with other reviewers.

Would you explain the rationale behind your proposal?